hadoop-tools/hadoop-azure/src/main/java/org/apache/hadoop/fs/azurebfs/AbfsStatistic.java

```diff
@@ -73,7 +73,36 @@
   READ_THROTTLES("read_throttles",
       "Total number of times a read operation is throttled."),
   WRITE_THROTTLES("write_throttles",
-      "Total number of times a write operation is throttled.");
+      "Total number of times a write operation is throttled."),
+
+  //OutputStream statistics.
+  BYTES_TO_UPLOAD("bytes_upload",
```

Comment:
these should all be in hadoop-common *where possible*, so that we have consistent names everywhere for aggregation

Reply:
Yes. I think apart from readAhead counter we can have all the others in common.